package.json

```diff
@@ -4,7 +4,10 @@
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react-hooks": "^3.2.1",
     "@testing-library/user-event": "^7.1.2",
+    "enzyme": "^3.11.0",
```

Comment:
Do you want to use Enzyme or the react-testing-library? I may be wrong but I don't think you need both. Whatever you choose - the testing frameworks just need to be in the devDependencies as we won't need them in production - just testing before we get to production.